src/detect-engine-tag.c

```diff
@@ -263,7 +263,7 @@ static void TagHandlePacketFlow(Flow *f, Packet *p)
 
     while (iter != NULL) {
         /* update counters */
-        iter->last_ts = SCTIME_SECS(p->ts);
+        iter->last_ts = (uint32_t)SCTIME_SECS(p->ts);
```

Comment:
@jlucovsky you worked on this `SCTIME_SECS` I guess.
But there are lots of places where this was silently downcast to a u32

What is the right solution ?